webpack.config.analyzer.js

```diff
@@ -0,0 +1,6 @@
+import {BundleAnalyzerPlugin} from 'webpack-bundle-analyzer';
+import config from './webpack.config.dev';
+
+config.plugins.push(new BundleAnalyzerPlugin());
```

Comment:
Just realized, would suggest moving the content of this file to tools/analyzeBundle.js. I don't see a need for this being a separate file. Right?